kuma/static/styles/components/payments/banner.scss

```diff
@@ -118,12 +138,75 @@
         box-sizing: border-box;
         margin: 0 auto;
         max-height: 100%;
-        max-width: 1320px;
-        padding: 28px;
+        max-width: $payment-banner-max-width;
+        padding: $payment-banner-padding;
```

Comment:
Kudos on replacing all of these with variables 👍

Reply:
Thanks! Slowly trying to refactor this feature so it's a little more manageable.